.github/workflows/lint.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: '3.x'
+          python-version: '3.9'
```

Comment:
Should be fixed on all linting?

Reply:
Perhaps, we should probably open an issue / PR and discuss it as sometimes it is nice for it to be latest but a fixed one for linting is probably okay.